### ISSUE TITLE
Support drag and drop note/file link

### ISSUE THREAD

**Issue by wenlzhang** [GitMurf/obsidian-drag-and-drop-blocks#8]
It appears that it already supports drag and drop note header. Based on this, would it be possible to drop note/file link? Specifically, one can use a different hotkey or paste with another option, to be able to paste only the note/file link, without header included?

**Comment by GitMurf** [comment]
So you mean if I drag any block from a page it will just simply copy the [[page name]] link to that page? So every block on the page would drag the same thing, right?

**Comment by wenlzhang** [comment]
Yes. One can drag and drop a link to the `[[page name]]` no matter which block it is or no matter which position it is on the page.
